plasmapy/analysis/nullpoint.py

```diff
@@ -1154,18 +1155,57 @@ def in_bound(pos):
     return null_point_find(**null_point_args)
 
 
+def _vspace_iterator(vspace, MAX_ITERATIONS=500, err=1e-10):
+    r"""
+    Returns an array of null point object, representing
+    the null points of the given vector space.
+
+    Parameters
+    ----------
+    vspace: array_like
+        The vector space as constructed by the vector_space function which is
```

Comment:
```suggestion
        The vector space as constructed by the ``_vector_space`` function which is
```
...since we made it private earlier.